tests/installation/product_selection/install_SLES.pm

```diff
@@ -12,7 +12,7 @@ use warnings;
 sub run {
     my $product_selection = $testapi::distri->get_product_selection();
 
-    $product_selection->wait_product_selection_page({timeout_scale => get_var('TIMEOUT_SCALE', 1), message => 'Product Selection page is not displayed'});
+    $product_selection->wait_for({message => 'Product Selection page is not displayed', waitpage => 'ProductSelectionPage', timeout => 10});
```

Comment:
we cannot pass the page name here, we are coupling the test layer with the specific implementation, we just pass the timeout and the timeout_scale.